### ISSUE TITLE
arm support?

### ISSUE THREAD

**Issue by AnotherDroog** [scaleway/image-alpine#64]
I would like to run this on C1 (armv7) or arm64 (armv8).

The repository had prior support for arm, it seems support was removed?

I would like to help out to fix it and would appreciate if you can point me in the right direction.

**Comment by JamesTheAwesomeDude** [comment]
Agreed, it's ridiculous to be loading a full Debian installation for something as resource-strapped as the $3/mo VPS

Tips for anyone starting work on this (I poked it a bit earlier today):

* The rescue boot mode boots you into an emergency SSH-enabled Ubuntu image, from which you can **completely format** `vda` if you so choose (the bootloader supports loading OS from a partitionless filesystem on this, too)
* Check around in the main repos for existing integration work, remember Alpine stock uses OpenRC
* Beware [this issue](https://github.com/scaleway/initrd/issues/202)
